### PR TITLE
 	Fixed bug 223, Insertions for VCF records. Added unit test.

### PR DESCRIPTION
**Pull request by nkindlon**

### CODE REVIEW

**Comment by marghoob** [comment]
Thanks for fixing this so quickly. By the way, just to avoid confusion, it would be good to change SVTYPE to INS in the test files where <INS> is used as the ALT.

**Comment by nkindlon** [comment]
Sorry, I'm a bit confused; the two files I added for test #47 in the unit test are bug223_sv1_a.vcf and bug223_sv1_b.vcf. They each have one INS and one DEL in their ALT fields. What else needs to be changed?

File: test/intersect/bug223_sv1_a.vcf [hunk 1]
```diff
@@ -0,0 +1,3 @@
+##fileformat=VCFv4.1
+chr1	1	a	G	<DEL>	70.90	.	TOOL=LUMPY;SVTYPE=DEL;SVLEN=-389,-4611;END=253195;STR=+-:4;IMPRECISE;CIPOS=-2,137;CIEND=0,0;EVENT=791255;SUP=4;PESUP=4;SRSUP=0;EV=PE;PRIN;CSQ=intergenic_variant||||||||||
+chr1	4	a	G	<INS>	70.90	.	TOOL=LUMPY;SVTYPE=DEL;SVLEN=-4611;END=253195;STR=+-:4;IMPRECISE;CIPOS=-2,137;CIEND=0,0;EVENT=791255;SUP=4;PESUP=4;SRSUP=0;EV=PE;PRIN;CSQ=intergenic_variant||||||||||
```
Comment (marghoob) [inline test/intersect/bug223_sv1_a.vcf]: @nkindlon I was referring to the INFO column here. Although the ALT field has `<INS>`, the INFO field has `SVTYPE=DEL` which can be confusing.

File: test/intersect/bug223_sv1_b.vcf [hunk 2]
```diff
@@ -0,0 +1,3 @@
+##fileformat=VCFv4.1
+chr1	5	a	G	<INS>	70.90	.	TOOL=LUMPY;SVTYPE=DEL;SVLEN=500;END=253195;STR=+-:4;IMPRECISE;CIPOS=-2,137;CIEND=0,0;EVENT=791255;SUP=4;PESUP=4;SRSUP=0;EV=PE;PRIN;CSQ=intergenic_variant||||||||||
```
Comment (marghoob) [inline test/intersect/bug223_sv1_b.vcf]: @nkindlon Another case where the INFO column is inconsistent with the ALT.

**Comment by nkindlon** [comment]
Ahhh, thank you. You're absolutely correct. We should make those consistent both for any future code changes and human readability. I'll change those now.

**Comment by marghoob** [comment]
Another point I noticed was the SVLEN field was negative for some
insertions-they should be made positive to adhere to the standard
conventions. Thanks for incorporating these changes.

On Thu, Apr 23, 2015 at 12:16 PM, Neil Kindlon <notifications@github.com>
wrote:

> Ahhh, thank you. You're absolutely correct. We should make those
> consistent both for any future code changes and human readability. I'll
> change those now.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/arq5x/bedtools2/pull/224#issuecomment-95689745>.
>



-- 

<http://www.bina.com>

*Marghoob Mohiyuddin, PhD  *//  Sr. Bioinformatics Scientist, Bina
Technologies, part of Roche Sequencing
E. marghoob@bina.com

*Learn More*  //  video <http://www.bina.com/product.html>  //  news
<http://www.bina.com/press-coverage.html>  //  posters
<http://f1000.com/posters/search?query=bina+technologies>  //  science
<http://www.bina.com/science.html>  //  platform
<http://www.bina.com/platform.html>
*Test Drive Bina  //  Experience NextGen analysis for yourself *
<http://www.bina.com/test-drive.html>

**Comment by nkindlon** [comment]
Also a good point. For that matter, I'd also just noticed the rest of the information in the tag line, specifically the number after the "END=" string, isn't right either.

**Comment by nkindlon** [comment]
Put changes in pull request #225.

**Pull request merged**